xcode/Subconscious/Shared/Components/Common/EntryRow.swift

```diff
@@ -18,11 +18,9 @@ struct EntryRow: View {
     var body: some View {
         VStack(alignment: .leading, spacing: AppTheme.unit) {
             SubtextView(
+                peer: entry.toPeer(),
```

Comment:
Not sure what the right name for this parameter is. We want the peer so we can selectively rebase depending on whether we tapped a link, a transclude or a link _within_ a transclude and provide a simple interface to the caller.

Reply:
fwiw this is why there were two callbacks passed before, to handle these two cases, but I think this approach is better.

Reply:
@bfollington question about conceptual approach... Since transcludes are always embedded within a sphere, and links are always embedded within a document or a transclude, shouldn't we be able to rebase at each level and present the same interface to consumers?

E.g. Alice transcludes Bob's note which has a link to Carol. The rebased Carol link, as exposed via `onLink` is `@carol.bob.alice/foo`.

Reply:
Separately, adding a `peer` property to SubtextView seems good! Subtext is always going to belong to some sphere, and because petnames are relative, linking is always relative. We should probably get links out of Subtext that point to the correct sphere, by default, and passing in a peer in forces correctness.